praktikumsplaner-frontend/frontend/src/views/MeldungPraktikumsstelle/MeldungAusbildung.vue

```diff
@@ -225,10 +226,42 @@ const requiredRule = [validationRules.notEmptyRule("Darf nicht leer sein!")];
 const emailRule = [
     validationRules.notEmptyRule("Darf nicht leer sein!"),
     validationRules.emailRule("Keine gültige Email"),
+    validationRules.maxLengthRule(
+        255,
+        "Die Email darf nicht länger als 255 Zeichen sein"
+    ),
 ];
```

Comment:
Our MailRule is not yet correct. The @Email annotation in the backend blocks writing to the database. 
In the frontend, however, mails are still accepted which are not possible - for example: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa@aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.de" is possible according to the frontend mask. However, this is not accepted by the backend. 
[Hibernate Email Annotation](https://github.com/hibernate/hibernate-validator/blob/main/engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/AbstractEmailValidator.java)